packages/common/src/schemas/sign-on/confirmEmailSchema.ts

```diff
@@ -0,0 +1,30 @@
+import { z } from 'zod'
+
+const messages = {
+  required: 'Code required.',
+  invalid: 'Invalid code.'
+}
+
+export const confirmEmailSchema = z.object({
+  otp: z
+    .string({ required_error: messages.required })
+    .regex(/^\d{3} \d{3}$/, { message: messages.invalid })
+})
+
+export const formatOtp = (inputString: string) => {
+  // Remove non-numeric characters
+  const numericOnly = inputString.replace(/\D/g, '')
+
+  // Limit the string to 6 characters
+  const limitedString = numericOnly.slice(0, 6)
+
+  // Insert a space after the third character if the string is more than three characters long
+  const formattedString = limitedString.replace(
+    /^(.{3})(.{0,3})$/,
+    (_, firstThree, rest) => (rest ? `${firstThree} ${rest}` : firstThree)
+  )
```

Comment:
do we need to actually insert a space in the formatted string here vs just displaying one to the user?

Reply:
on second thought, this is probably fine bc we strip out the space later anyway and this is probably easier than custom fake formatting logic

Reply:
Yeah we could go a few different ways. This just made the verification code look consistent from email to app